utilities/copyoffload_migration.py

```
@@ -215,3 +212,81 @@ def _check_file() -> bool:
                 LOGGER.warning(f"Failed to power off VM '{vm_name}': {type(e).__name__}: {e}")
         else:
             LOGGER.info(f"Leaving VM {vm_name} powered on")
+
+
+def verify_xcopy_used(
+    ocp_admin_client: DynamicClient,
+    plan: Plan,
+    target_namespace: str,
+    expected_xcopy_used: bool,
```

Ruff (0.15.6)

[warning] Boolean-typed positional argument in function definition

(FBT001)